src/styles/blocks/navigation.scss

```diff
@@ -0,0 +1,46 @@
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+
+  .nav__item {
+    display: flex;
+    position: relative;
+    margin-left: 20px;
+    height: 60px;
+
+    &:first-child {
+      margin-left: 0;
+    }
+
+    .nav__link-item {
```

Comment:
You don't need to nest selectors like this as in this case, you have a selector like this:
<img width="263" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/63243136/3d3d6d00-7ba5-4922-a87a-6486ef46388f">

They all just should be inside `nav`
```scss
.nav {
  &__list {}

  &__item {}
  // other
}
```